pyprof/nvtx/nvmarker.py

```diff
@@ -273,31 +279,39 @@ def add_wrapper(mod, fn_name):
                                        ) and (type(mod) is not torch.jit.TopLevelTracedModule)
 
     def wrapper_func(*args, **kwargs):
+        global wrappers_enabled
```

Comment:
Not a huge fan of using global variables. Was this not possible by passing an argument to the call?

Reply:
I don't think it is possible. We don't "call" the wrapper function. It is injected via monkey patching.

Reply:
Yea that makes sense. It acts more like a pointer in a vtable that we overwrote vs a function call.